Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.2
+
+RUN echo 'http://dl-4.alpinelinux.org/alpine/edge/main' >> /etc/apk/repositories \
+    && apk update && apk add go git ca-certificates make && rm -rf /var/cache/apk/* \
+    && wget -P /usr/local/bin http://public.thisissoon.com.s3.amazonaws.com/glide \
+    && chmod +x /usr/local/bin/glide
+
+ENV GOPATH=/deadpool
+
+RUN go get github.com/gorilla/mux
```

Comment:
I'd certainly look at using `gpm` as we do on Myleene and other go projects to manage dependencies during the build.